jupyter-book/predictive_modeling_pipeline/03_categorical_pipeline_video.md

```diff
@@ -0,0 +1,3 @@
+# 🎥 How to define a scikit-learn pipeline and visualize it
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/5ri7BhFrNe4?rel=0" title="YouTube video player" frameborder="0" rel="0" showinfo="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Let's wrap the player in a dedicated div with a specific CSS class so that we can get a better control in the CSS:

```suggestion
<div class="video_player">
<iframe width="640" height="480" src="https://www.youtube.com/embed/5ri7BhFrNe4?rel=0" title="YouTube video player" frameborder="0" rel="0" showinfo="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</div>
```

also, let's restore conventional dimensions that are only properly taken into account once the CSS that was meant to be applied to the slides iframes are disabled in `sklearn_mooc.css`.